Tasks/XcodePackageiOS/xcodepkgios.ts

```diff
@@ -1,7 +1,7 @@
 import tl = require('vsts-task-lib/task');
 import path = require('path');
 
-var sourceDir = tl.getVariable('build.sourcesDirectory');
+var sourceDir = tl.getVariable('build.sourcesDirectory') || tl.getVariable('System.DefaultWorkingDirectory');
```

Comment:
I assume this is a well known build variable? I don't see it as a env var on Windows.